_rules/SC2-4-2-page-has-title.md

```diff
@@ -119,6 +119,18 @@ Valid `title` provided.
 </html>
 ```
 
+#### Passed example 6
+
+`title` element does not only contain whitespace.
+
+```html
+<html>
+  <body>
+    <title>:-)</title>
```

Comment:
This should fail WCAG, I don't think we should put that into a "passed" example.